docs/Versioning.md

```diff
@@ -90,9 +77,12 @@ ONNX specification. The union of the operator sets specified by a given model MU
 
 How nodes bind to operator declarations is strictly defined and are designed to increase model compatibilitey across ONNX implementations (appealing to the conservative clause of the robustnes principle). 
 
-How ONNX implementations bind an operator declaration to specific implementation is outside the scope
-of this specification.  Implementations of ONNX MAY elect 
-to introduce more sophisticated operator declaration/implementation binding modes to appeal to the liberal clause of the robustness principle. 
+How ONNX implementations bind an operator declaration to specific implementation is outside the scope of this specification.
+Implementations of ONNX MAY elect to introduce more sophisticated operator declaration/implementation binding modes to appeal to the liberal clause of the robustness principle.
+
+Implementations and models MAY provide experimental operators by agreeing to use operators in an experimental domain.
+
+Implementations and models MAY provide experimental IR features by agreeing to use fields from reserved ranges.
```

Comment:
Not sure what this means in practice. Experimental ops are documented with the regular ops. where are these experimental fields documented? How do we avoid conflicts between different experimenters?

Reply:
See above. I mixed versioning with extending here, I'm removing that verbiage.